src/utils.js

```diff
@@ -0,0 +1,4 @@
+// Creates a handler from the given function
+export function handler(handle) {
+  return { handle };
+}
```

Comment:
Don't we have that one yet somewhere? It just looks very familiar.

Reply:
This was refactored out from `defaultHandlers.js` - I moved it to a separate file to avoid circular imports.

Reply:
Makes sense; perhaps `handlerUtil.js` then or so? (Your call.)